src/index.js

```diff
@@ -18,34 +18,47 @@ function resolve (dir) {
 
 /**
  * Create a webpack config usable for demosplan-core addons.
- * 
+ *
  * ## Usage
- * 
+ *
  * ```js
  * const DemosplanAddon = require('@demos-europe/demosplan-addon')
- * 
+ *
  * module.exports = DemosplanAddon.build(
- *   'my-addon-name', 
+ *   'my-addon-name',
  *   { 'MyAddon': DemosPlanAddon.resolve('src/index.js') }
  * )
  * ```
- * 
+ *
  * @param {String} addon_name the name, d'uh
- * @param {Object} entrypoints name-mapped entry points dictionary 
+ * @param {Object} entrypoints name-mapped entry points dictionary
  * @returns {Options} webpack configuration
  */
 function configBuilder(addon_name, entrypoints) {
   const isProduction = process.env.NODE_ENV == 'production'
 
+  /**
+   * Transform the entry points object into a usable format
+   * so a browser is able to digest the output but still keep
+   * the simple webpack config mentioned above to create addons
+   */
+  for (const key of Object.keys(entrypoints)) {
+    entrypoints[key] = {
+      import: entrypoints[key]
+    }
+    entrypoints[key]['library'] = {
+      name: key,
+      type: 'window'
+    }
+  }
+
   return {
     entry: entrypoints,
     mode: isProduction ? 'production' : 'development',
     output: {
       path: resolve('dist'),
       filename: `[name].umd.js`,
-      library: `${addon_name}`,
-      libraryTarget: 'umd',
-      libraryExport: 'default'
+      library: `${addon_name}`
```

Comment:
Nitpicky question: wouldn't it be enough to assign the string like so?

```suggestion
      library: addon_name
```